src/js/common/helpers/login-helpers.js

```diff
@@ -39,6 +39,8 @@ export function getUserData(dispatch) {
       dispatch(updateProfileField('status', json.data.attributes.va_profile.status));
       dispatch(updateProfileField('services', json.data.attributes.services));
       dispatch(updateLoggedInStatus(true));
+    } else {
+      dispatch(profileLoadingFinished());
```

Comment:
If we're going to have a new action, then can it also be dispatched above in the case where the response has user data? So that would dispatch both `updateLoggedInStatus(true)` and `profileLoadingFinished()` and below  in src/js/user-profile/reducers/profile.js you wouldn't have to set `loading` to false for an `UPDATE_LOGGEDIN_STATUS` action.